src/components/structures/LoggedInView.tsx

```diff
@@ -502,7 +503,10 @@ class LoggedInView extends React.Component<IProps, IState> {
                 handled = true;
                 break;
             case NavigationAction.ToggleShortCutDialog:
-                KeyboardShortcuts.toggleDialog();
+                dis.dispatch<OpenToTabPayload>({
+                    action: Action.ViewUserSettings,
+                    initialTabId: UserTab.Keyboard,
+                });
```

Comment:
The shortcut is called *toggle* but no longer seems to toggle

Reply:
Yeah, though I am wondering if it should be that way, now that it is a settings tab, will leave it up to design to choose what is ideal here

Reply:
Needs input from @matrix-org/design

Reply:
I think just changing it to `OpenShortcutDialog` is sane (and the accompanying copy also)